cypress/support/fragments/finance/financeHelper.js

```diff
@@ -51,3 +59,35 @@ export default {
     cy.do(Button({ icon: 'times' }).click());
   }
 };
+
+Cypress.Commands.add('createFundViaUI', (fund) => {
```

Comment:
I suggest don't use custom cypress steps into fragments.

Reply:
Where should I put it?

Reply:
you can define method and use it in fragment as usual.

Reply:
Done